package.json

```diff
@@ -43,14 +43,12 @@
   },
   "dependencies": {
     "@stoplight/json": "^3.10.0",
-    "@stoplight/json-schema-merge-allof": "^0.7.2",
+    "@stoplight/json-schema-tree": "1.0.0-beta.5",
```

Comment:
Update this to `1.0` and the functionality in this PR looks good to me! It's a bit difficult to review, but high level looks good and we reviewed some things on last week's call. It's going into beta branch so we can iron out any little issues that shake out over next few weeks IMHO.